recipes/uwsgi/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "2.0.12" %}
+
+package:
+  name: uwsgi
+  version: '{{ version }}'
+
+source:
+  fn: "uwsgi-{{ version }}.tar.gz"
+  url: "https://pypi.io/packages/source/u/uWSGI/uwsgi-{{ version }}.tar.gz"
+  md5: 1451cab954bad0d7d7429e4d2c84b5df
+  patches:
+    - darwin.patch   # [osx]
+    - darwin2.patch  # [osx]
+    # ^^ patch darwin2 should be able to be deleted on the next version pump
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+    skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - gcc      # [unix]
```

Comment:
So, we are not in the habit of using `gcc`. It is actually frowned upon. This is partially due to the fact that system compilers end up being very effective for the most part. It is also partially due to issues that have been encountered by packaging and distributing `libgcc`.